build.sbt

```diff
@@ -47,123 +46,109 @@ lazy val commonSettings = Seq(
   pomExtra := pomInfo,
   resolvers += Opts.resolver.mavenLocalFile,
   autoAPIMappings := true,
-  libraryDependencies ++= excludeServlet(Seq(
-    "org.scalaz" %% "scalaz-core" % "7.1.3",
-    "org.mortbay.jetty" % "servlet-api" % "3.0.20100224" % "provided",
```

Comment:
Deleted for the same reason as above, no need for us to be doing this - we don't use it!